boards.txt

```diff
@@ -39802,3 +39802,61 @@ alfredo-nou3.menu.EraseFlash.all=Enabled
 alfredo-nou3.menu.EraseFlash.all.upload.erase_cmd=-e
 
 ##############################################################
+codecell.name=CodeCell
+codecell.vid.0=0x303a
+codecell.pid.0=0x1002
+codecell.upload_port.0.vid=0x303a
+codecell.upload_port.0.pid=0x1002
+
+codecell.bootloader.tool=esptool_py
+codecell.upload.tool=esptool_py
+codecell.upload.maximum_size=4194304
+codecell.upload.maximum_data_size=327680
+codecell.upload.use_1200bps_touch=false
+codecell.upload.wait_for_upload_port=false
+
+codecell.upload.speed=921600
+codecell.serial.disableDTR=false
+codecell.serial.disableRTS=false
+
+codecell.build.tarch=riscv32
+codecell.build.target=esp
+codecell.build.mcu=esp32c3
+codecell.build.core=esp32
+codecell.build.variant=codecell
```

Comment:
The variant name is not matching the variant folder. Please change the variant folder to lowercase.